docs/plans/2026-05-03-matrix-factorization-implementation.md

```
@@ -0,0 +1,243 @@
+- `R`: Matriz de ratings usuario-item (m × n)
+- `U`: Matriz de usuarios (m × k)
+- `Σ`: Valores singulares diagonales (k × k)
+- `V^T`: Matriz de items transpuesta (k × n)
+
+Despues de reduccion a k factores:
+- `userFactors = U_k × sqrt(Σ_k)` (vectores latentes de usuarios escalados equitativamente)
+- `itemFactors = V_k × sqrt(Σ_k)` (vectores latentes de items escalados equitativamente)
+
+### Prediccion
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)